jwst/ramp_fitting/ramp_fit.py

```diff
@@ -1009,15 +999,15 @@ def dq_compress_sect(gdq_sect, pixeldq_sect):
 
     Parameters
     ----------
-    gdq_sect: int (uint8), 3D array
+    gdq_sect : int (uint8), 3D array
         cube of GROUPDQ array for a data section
 
-    pixeldq_sect: int (uint16), 2D array
+    pixeldq_sect : int (uint16), 2D array
```

Comment:
Is the data type of this array still uint16? The pixeldq array is now up to uint32. Maybe just list it as "int" and don't worry about the specific size.